### DISCUSSION TITLE
Internal mail (from cron etc.) redirection

### DISCUSSION THREAD

**Discussion by salixh5** [simple-login/app#837] [Self-hosting question]
This is not really a huge issue, but I'm still a Postfix noobie so for me it's uncharted territory. Sometimes I see mailing attempts in the logs like these:
```
Mar 10 08:22:07 mail postfix/pickup[152952]: 15A55A04D8: uid=1000 from=<myusername>
Mar 10 08:22:07 mail postfix/cleanup[153361]: 15A55A04D8: message-id=<20220310072207.15A55A04D8@mail.mydomain>
Mar 10 08:22:07 mail postfix/qmgr[9204]: 15A55A04D8: from=<myusername@mydomain>, size=585, nrcpt=1 (queue active)
Mar 10 08:22:07 mail postfix/smtp[153365]: 15A55A04D8: to=<myusername@mydomain>, orig_to=<myusername>, relay=127.0.0.1[127.0.0.1]:20381, delay=0.24, delays=0.07/0.05/0.01/0.11, dsn=5.0.0, status=bounced (host 127.0.0.1[127.0.0.1] said: 550 SL E3 Email not exist (in reply to end of DATA command))
Mar 10 08:22:07 mail postfix/bounce[153367]: 15A55A04D8: sender non-delivery notification: 4E269A2325
Mar 10 08:22:07 mail postfix/qmgr[9204]: 15A55A04D8: removed
```
And corresponding to this:
```
Mar 10 08:22:07 mail postfix/cleanup[153361]: 4E269A2325: message-id=<20220310072207.4E269A2325@mail.mydomain>
Mar 10 08:22:07 mail postfix/qmgr[9204]: 4E269A2325: from=<>, size=2558, nrcpt=1 (queue active)
Mar 10 08:22:07 mail postfix/smtp[153365]: 4E269A2325: to=<myusername@mydomain>, relay=127.0.0.1[127.0.0.1]:20381, delay=0.1, delays=0/0/0/0.09, dsn=5.0.0, status=bounced (host 127.0.0.1[127.0.0.1] said: 550 SL E3 Email not exist (in reply to end of DATA command))
Mar 10 08:22:07 mail postfix/qmgr[9204]: 4E269A2325: removed
```
This is probably something like cron or the like that is trying to send mail to itself I would reckon. Should I just ignore these or is there some way of disabling these or maybe collecting them all in one mailbox that actually exists? Curious.

**Comment by nguyenkims** [comment]
This error `550 SL E3 Email not exist` means the alias doesn't exist.

**Reply by salixh5** [reply]
Yes, sorry if the question wasn't clear:

Let's say my Unix username is called "myusername" -- some programs running on the machine might try to send mail to this user locally. Postfix will pick it up and send it to SimpleLogin like any other mail. The alias "myusername" or "myusername@mydomain" does indeed not exist.

What I was trying to understand is how to best deal with this:
* should I setup an alias for every unix user on the machine to be able to receive all mails?
* is there another way to automatically get all local mail sent to me?
* if not one of the above, could I at least configure postfix so that it doesn't forward the local mail to SimpleLogin? After all, it's local mail that's not really intended to be handled by SimpleLogin in the first place. It should just get dropped or maybe written to a file.

**Reply by nguyenkims** [reply]
I think the 3rd approach is maybe the solution. Postfix supports multi instance http://www.postfix.org/postmulti.1.html and maybe you can have the default postfix forwarding the emails to SimpleLogin and another Postfix instance receiving the system emails. In this case you might need to tell the system to not use another port rather than the default 25.
